### PR TITLE
PHP Date and Time

### LINKED ISSUE #7882: [Backend Development with PHP] : PHP Date and Time
### Description

Documentation on PHP Date and Time

### Domain

Backend Dev (PHP)

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by palak1101**
<hr>

## Description 📜

This documentation on PHP Date and Time includes a brief explanation on date(), time() and other functions along-with examples code and its output. Also, a real time application of date and time function is described.

Fixes #7882

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [x] Backend Dev (PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by palak1101** [comment]
@praveenscience Sir, please review this PR.

File: Backend_Development_With_PHP/Coding_Practices/Date_And_Time/DateandTime.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# PHP : Date and Time
+
+Dates are very important part of everyday life with the help of which we are able to track our work regularly without even thinking. Some of the most used operations in PHP while designing a website or SQL query execution are date and time. PHP provides powerful tools for date arithmetic that makes it easy to manipulate date. Some of the predefined functions in PHP for date and time are discussed below.
+
+## PHP date() Function   
+
+The PHP date() function is used to convert a timestamp into more readable date and time format. The computer stores dates and times in a format called UNIX Timestamp, which measures time as number of seconds since the beginning of the Unix epoch (midnight Greenwich Mean Time on January 1, 1970 i.e. January 1, 1970 00:00:00 GMT ). Since it is not possible for a human to resd or understand this, PHP converts a timestamp in a format which is more understandable by human.
```
Comment (meinhoonharsh) [inline Backend_Development_With_PHP/Coding_Practices/Date_And_Time/DateandTime.md:7]: would be better if we've date() and other functions under backticks like `date()` through \`date()\`

**Comment by palak1101** [comment]
@meinhoonharsh I've added the changes suggested by you, please review it.

**Review by meinhoonharsh — APPROVED** [review]
LGTM

**Review by praveenscience — APPROVED** [review]
👍

**Pull request merged**